### PR TITLE
Sean/more achievements

### PR DESCRIPTION
**Pull request by seangeggie**
Added a few new achievements. Added some parameters to win and report behaviours to support them.

### CODE REVIEW

File: tourney/achievements/achievements.py [hunk 1]
```diff
@@ -14,6 +14,11 @@
 from .flawless_defeat_achievement import FlawlessDefeatAchievement
 from .reporter_achievement import ReporterAchievement
 from .leave_channel_achievement import LeaveChannelAchievement
+from .firstJoinerAchievement import FirstJoinerAchievement
```
Comment (netromdk) [inline tourney/achievements/achievements.py]: I'd like the file name to use the same pythonic way as the others: `first_joiner_achievement`

File: tourney/achievements/threeplayer_win_achievement.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from .tiered_achievement import TieredAchievement
+from .behavior import WIN_BEHAVIOR
+
+class ThreePlayerWinAchievement(TieredAchievement):
+  def __init__(self):
+    tiers = (
+      (1,   "Three Amigos", "Win a match in a three-player team."),
+      (3,   "Three Musketeers", "Win three matches in three-player teams."),
+      (33,  "Three Kings", "Win thirty three matches in three-player teams."),
+      (333, "Holy Trinity", "Win three hundred and thirty three matches in three-player teams."),
```
Comment (netromdk) [inline tourney/achievements/threeplayer_win_achievement.py]: This is great :p

File: tourney/achievements/threeplayer_win_achievement.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  def update(self, behavior):
+    user_id = behavior.user_id()
+    user_team = behavior.winner_team()
+    self.__check_init(user_id)
+    if len(user_team) == 3:
+      self.data[user_id][0] += 1
+      amount = self.data[user_id][0]
+      nt = self.next_tier(user_id)
+      if amount == nt or amount == nt+1:
```
Comment (netromdk) [inline tourney/achievements/threeplayer_win_achievement.py]: We don't needs this check for matches won, it's only for counting rounds that can be 1 or 2.

File: tourney/achievements/threetwo_win_achievement.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+from .tiered_achievement import TieredAchievement
+from .behavior import WIN_BEHAVIOR
+
+class ThreeVTwoWinAchievement(TieredAchievement):
+  def __init__(self):
+    tiers = (
+      (1,   "Strength in Numbers", "In a three-player team, beat a two player team."),
+      (10,  "E Pluribus", "In a three-player team, beat a two player team ten times."),
+      (50,  "My Name is Legion", "In a three-player team, beat a two player team fifty times."),
```
Comment (netromdk) [inline tourney/achievements/threetwo_win_achievement.py]: Maybe "I am Legion"?

File: tourney/achievements/threetwo_win_achievement.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  def update(self, behavior):
+    user_id = behavior.user_id()
+    user_team = behavior.winner_team()
+    loser_team = behavior.loser_team()
+    self.__check_init(user_id)
+    if len(user_team) == 3 and len(loser_team) == 2:
+      self.data[user_id][0] += 1
+      amount = self.data[user_id][0]
+      nt = self.next_tier(user_id)
+      if amount == nt or amount == nt+1:
```
Comment (netromdk) [inline tourney/achievements/threetwo_win_achievement.py]: We don't needs this check for matches won, it's only for counting rounds that can be 1 or 2.

File: tourney/achievements/achievements.py (hunk 1 above)
Comment (seangeggie) [reply tourney/achievements/achievements.py]: Yeah, good point.

File: tourney/achievements/threeplayer_win_achievement.py (hunk 3 above)
Comment (seangeggie) [reply tourney/achievements/threeplayer_win_achievement.py]: Oh yeah! True! Copy-paste :P

File: tourney/achievements/threetwo_win_achievement.py (hunk 4 above)
Comment (seangeggie) [reply tourney/achievements/threetwo_win_achievement.py]: https://www.biblegateway.com/passage/?search=Mark%205:9&version=NIV but yeah, that form is probably more widely recognized.

Comment (netromdk) [reply tourney/achievements/threetwo_win_achievement.py]: Oh yeah, I like your version better actually.

**Comment by seangeggie** [comment]
Changes made 👍

**Comment by netromdk** [comment]
Thanks!

**Pull request merged**